openwisp_firmware_upgrader/templates/admin/upgrade_selected_confirmation.html

```diff
@@ -52,7 +52,9 @@ <h2>{% trans "Devices which the system has never upgraded yet" %} ({{ firmwarele
   <input type="submit" name="upgrade_related" value="{% trans "Upgrade only related devices" %}">
   {% endif %}
   {% endif %}
+  {% if related_count or firmwareless_count %}
```

Comment:
So if there's no devices, how do the users go back? Maybe we could still provide the button but change its anchor text to show something like:

```
No devices to upgrade

<link>Go back</link>
```

Reply:
In my case I just clicked on the back button of my browser but I like the idea of having an extra button saying "Go back"